pyne/mesh.py

```diff
@@ -1510,6 +1511,102 @@ def _do_op(self, other, tags, op, in_place=True):
 
         return mesh_1
 
+
+class MeshTally(StatMesh):
+    """This class stores all information from all single mesh tally that
+    exists within some meshtal or state point file. Header information is
+    stored as attributes and the "mesh" attribute is a MOAB mesh with all
```

Comment:
Technically a PyNE Mesh.
```suggestion
    stored as attributes and the "mesh" attribute is a PyNE mesh object with all
```